blade/procData/models.py

```diff
@@ -11,4 +11,17 @@ class Meta:
 		ordering = ['-id']
 
 	def __str__(self):
-		return self.email
\ No newline at end of file
+		return self.email
+
+
+#class results
+
+#class query
+class dbQueriedUsers(models.Model):
```

Comment:
I would name it QueriesUsers (don't change it anymore though, fine for now)

More importantly, did you generate the migration file with `python manage.py makemigrations`?